extensions/config-viewer/runtime/src/main/java/io/quarkus/config/ConfigHolder.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.config;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.Config;
+
+public class ConfigHolder {
```

Comment:
This shouldn't be needed IIUC, you could just pull `Config` out of CDI.current() instead of wrapping in this class

Reply:
Ok, I'll try to simplify that.

Reply:
Removed `ConfigHolder`.